### DISCUSSION TITLE
2023-05-03 Weekly Meeting 106 Notes/Agenda

### DISCUSSION THREAD

**Discussion by funksoup** [omigroup/omigroup#387] [General]
Weekly Meeting 106 is happening on Wednesday, May 3, 2023 from 9:00 AM PST (16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

UPDATES:

MSF-delegates:

- There is a board election coming up - it's important that we/OMI have a representative on the MSF board, so we can continue to be a voice for independent creators, small orgs, open source and community/grassroots orgs. Elections will be in a few weeks but we can start preparing our community for that now. (There are some documents we will need to fill out).

- Interoperability issue to bring to the attention of MSF and industry in general (via @aaronfranke):
-- There are competing standards between OMI's [OMI_collider](https://github.com/omigroup/gltf-extensions/tree/main/extensions/2.0/OMI_collider) and [OMI_physics_body](https://github.com/omigroup/gltf-extensions/tree/main/extensions/2.0/OMI_physics_body) glTF extensions and Microsoft's physics https://github.com/KhronosGroup/glTF/pull/2257.
-- Evo will interview @aaronfranke and then post the video so that we can get more engagement on this issue.
-- How can we get more engagement? Email the MSF general list (but make the email shorter - make ask clear)
-- Evo suggested Ben Houston at ThreeKit - maybe hit up specific people like this on LinkedIn?

-- Issue is described in detail in this [OMI_collider and OMI_physics_body pull request](https://github.com/KhronosGroup/glTF/pull/2258) and there's a bulleted list of specific differences and impacts. 

Media group: 

WORLD HOPS:
Proposal to do a metatraversal world hop that is a real world/virtual world hybrid event for [AWE](https://www.awexr.com/)? 
AWE conference is May 31-June 2 in Santa Clara, CA. If we can get some folks who will be onsite at AWE to help facilitate, we could do an OMI metatraversal world hop, similar to what we did at the OMI 100th meeting anniversary event. 

We could visit / meet in these spaces (via @madjin): 
https://hyperfy.io/msf
https://hyperfy.io/neonstreet
https://hyperfy.io/omi
https://framevr.io/omi

-- Evo said she discussed doing the World Hop with Ben Irwin -- maybe between May 16-24 (the week before AWE). They are already talking about itinerary options. Ben can help with technical direction. Also talked with CodeFrau.

-- Evo is completing some music videos. She would like to include footage of the portal hop. Shooting dates? 

-- Need to confirm with @madjin on world hop dates - **Friday, May 19** suggested.

-- Keyframe previously mentioned he will be at AWE onsite, and will be giving a talk on ["Mapping the Web3 Immersive Platforms"](https://www.awexr.com/usa-2023/agenda/3658-mapping-the-web3-immersive-platforms) on May 31 at 5:35-6pm. He may be able to facilitate the world hop and possibly facilitate streaming his talk (into our immersive space and vice versa) as well. Evo will be onsite at AWE as well (and may be able to facilitate a world hop).

-- Keyframe will also be planning an event on May 24 - 'Workshopping the Metaverse' in Spatial. Will be an open format -- people invited to come and ask questions. Do we have OMI slides for him to use? (Check with @mrmetaverse).

- We will need folks to be welcomers / hosts that can facilitate people joining the world hop to meet & greet.

To follow up on organizing the proposed world hop for AWE, we can use our [pro temp events channel in Discord](https://discordapp.com/channels/770382203782692945/1052644259741843526).

Media group:
- Evo needs to pass her role on the media team to someone. @musicscene might be working on a newsletter?
- @JimmyDof would like to help with meet & greet for world hop but also in general, on a media planning committee for event production. @AvatarJoy @MalBurns as well.
- @mrmetaverse and anyone that has slides and media/PR materials - please drop it into the [github repo](https://github.com/omigroup/media)

GLTF group:
- @aaronfranke - finally merged GLTF extensions in Godot Engine (specifically, OMI collider and OMI physics body).
- Several months ago - worked on ability for user to script their own extensions - this is in the build.

**Comment by mrmetaverse** [comment]
What if we simplify our processes and goals. 

- # of Implementors of our protocols and open standards we evangelize. 
- drive towards more visibility and engagement from larger corps.

**Reply by mrmetaverse** [reply]
Jin asked: "what would incentivize people to implement or participate?" 

This resonates with the "empower the people making the metaverse" core value we have. What would drive engagement/adoption? 
- project admin stuff like taking notes 
- also implementors why should they care 
- raising awareness: do people even know about our work? Are we teaching people how to implement or contribute?

**Reply by mrmetaverse** [reply]
Funksoup also shared: Getting people excited promotes participation too. Having events, making it fun, connecting people etc. 

Jimmy6Dof  also shared that that is what brought him to OMI as well. Promoting community and collaboration and events together, etc.

**Comment by mrmetaverse** [comment]
Speaking of events: 

We have the Friday May 19th World hop coming up!

**Reply by mrmetaverse** [reply]
Two world hop sessions, 1 that is VR specific and the other that is not VR specific. 

Thinking 12p PST (19:00 UTC) 
and 5p PST (00:00UTC)

**Reply by mrmetaverse** [reply]
proposed RSVPs

**Comment by mrmetaverse** [comment]
# Media team 

Lets sync tomorrow or Friday! https://savvycal.com/mrmetaverse/chat

**Comment by mrmetaverse** [comment]
# Starboard! 
- you can star react anything 
- ![image](https://user-images.githubusercontent.com/63426722/235981143-7e895c87-4e0a-4e09-bb38-bd32ffee74ea.png)
- if it gets 2 votes it posts to the starboard channel
- ![image](https://user-images.githubusercontent.com/63426722/235981361-d3432f3c-6044-4b5b-847d-aa88b0ef8fc5.png)
- Settings example: 
![image](https://user-images.githubusercontent.com/63426722/235981293-46c25a60-5bac-4162-98a0-3a9eca113cb5.png)

**Comment by mrmetaverse** [comment]
# Tip bot! 

Now we can send tips, or pool pizza funds for AFK events, and more! It can convert currencies, including tokens like MATIC.

**Comment by mrmetaverse** [comment]
# For AI bots 

- Evo would like us to get a collaboration agreement in place for how we will use AI bots, or how things may spin out from our org, etc.

**Reply by mrmetaverse** [reply]
- like if we use OMI bots, we agree to some open source license of the outputs, etc.
- this also expands on our previous work on standards for how we work together, handle conflict, etc. 


We should have this done before 501c3 OpenCollective swithc.

**Comment by mrmetaverse** [comment]
# Summary of priorities right now: 
- [ ] 501c3 Opencollective is 1 page, quick win, top priority
- [ ] let's codify our working agreement and dispute resolution thing + using tools agreement (Friday champions discussion)
- [ ] MSF election is coming up 
- [ ] next World hop(s) on 5/19 
- [ ] "Our one metric that matters": How many implementers? (consquently, how do increase implementors... etc.) 
- [ ] Media team sync! Friday at 1pm PDT - 4pm EST
